gcc/rust/rust-lang.cc

```diff
@@ -172,38 +172,29 @@ static tree
 grs_langhook_type_for_mode (machine_mode mode, int unsignedp)
 {
   // TODO: change all this later to match rustc types
```

Comment:
hehe, we should definitely do this at some point

Reply:
Yes... At first I wanted to see if I could DTRW, but then figured it's a bit more involved than hacking a few lines...